README.md

```diff
@@ -1 +1,63 @@
-# better-unit
\ No newline at end of file
+# better-unit
+
+better-unit is a wrapper around the [nodeunit](https://www.npmjs.com/package/nodeunit) module, with the following additions:
+
+* Support for comparing `Map`, `Set`, `WeakMap`, and `WeakSet` using the `deepEqual()` assertion.
```

Comment:
So why not just add a note that they can be used separately from the promise api by just including the individual plugin files?